src/api/v1/history.ts

```diff
@@ -0,0 +1,55 @@
+import {APIClient} from '../client'
+
+import {
+    Checksum256,
+    Name,
+    NameType,
+    Int32,
+    UInt32,
+    UInt32Type,
+    PublicKeyType,
+    PublicKey, Int32Type,
+} from '../../chain'
+
+import {
+    GetActionsResponse,
+    GetTransactionResponse,
+    GetKeyAccountsResponse,
+    GetControlledAccountsResponse,
+} from './types'
+
+export class HistoryAPI {
+    constructor(private client: APIClient) {}
+
+    async get_actions(accountName: NameType, pos: Int32Type, offset: Int32Type) {
+        return this.client.call({
+            path: '/v1/history/get_actions',
+            params: { account_name: Name.from(accountName), pos: Int32.from(pos), offset: Int32.from(offset) },
+            responseType: GetActionsResponse,
+        })
+    }
+
+    async get_transaction(id: Checksum256, blockNumHint?: UInt32Type) {
```

Comment:
Our APIs have a traces: boolean that can be passed to get the response w/o the traces, we should expose that since we use it ourselves in unicove. I suggest a third param `excludeTraces?: boolean` that if true adds a `traces: false` to the params (but otherwise omits it as not to confuse any other nodeos variant)